### PR TITLE
Configure for Codecov

### PR DESCRIPTION
**Pull request by chadwhitacre**
Primary PR for #482! 💃 After #545.

## Before Deployment

- [x] Populate `GH_APP_PRIVATE_KEY_FOR_CODECOV` to GH prod environment.
- [x] Set a new webhook secret for getsantry ([race condition](https://github.com/getsentry/eng-pipes/pull/551#issuecomment-1650817303), derp), store it in 1password.
- [x] Configure webhook and webhook secret in CoveCod ... leave inactive.
 
## After Deployment

- [ ] [Activate webhook](https://github.com/organizations/codecov/settings/apps/covecod)!
- [ ] [Subscribe to events](https://github.com/organizations/codecov/settings/apps/covecod/permissions)!
- [ ] Try it out in `codecov/feedback`!

### CODE REVIEW

**Review by hubertdeng123 — APPROVED** [review]

File: github-orgs.yml [hunk 1]
```diff
@@ -47,3 +47,17 @@ getsentry:
...
+    privateKey: 'GH_APP_PRIVATE_KEY_FOR_CODECOV'
+    installationId: 40036835
+  project:
+    nodeId: 'PVT_kwDOAH2Fnc4ARzsq'
+    fieldIds:
+      productArea: 'PVTSSF_lADOAH2Fnc4ARzsqzgLXyZg'
+      status: 'PVTSSF_lADOAH2Fnc4ARzsqzgLXyI4'
+      responseDue: 'PVTF_lADOAH2Fnc4ARzsqzgLXyhE'
+  repos:
+    withoutRouting:
```
Comment (hubertdeng123) [inline github-orgs.yml:62]: note: once there are repos with routing there may be some issues

Comment (chadwhitacre) [reply github-orgs.yml:62]: Say more?

**Comment by chadwhitacre** [comment]
Private key and webhook are configured.

Screenshot 2023-07-26 at 11 05 34 AM

Screenshot 2023-07-26 at 11 00 22 AM

File: github-orgs.yml (hunk 1 above)
Comment (chadwhitacre) [reply github-orgs.yml:62]: Oh! Maybe this isn't what you mean, but we need to figure out Slack channel routing for notifications. 🤔

Back to #482 for that ...

Comment (hubertdeng123) [reply github-orgs.yml:62]: Yep, that's what I meant here. We will need to configure a channel to send notifications for support, unless we want to also use the `#discuss-support-open-source` for codecov too.

**Pull request merged**